src/pages/BrowserPage.qml

```diff
@@ -133,13 +133,21 @@ Page {
         height: browserPage.height
     }
 
+    // Handle input region when portrait
     InputRegion {
-        window: webView.chromeWindow
+        window: webView.portrait ? webView.chromeWindow : null
         y: webView.enabled && browserPage.active ? overlay.y : 0
         width: browserPage.width
         height: webView.enabled && browserPage.active ? browserPage.height - overlay.y : browserPage.height
     }
 
+    // Handle input region when landscape
+    InputRegion {
```

Comment:
How about inverted landscape?

Would it be cleaner to move this handling to C++ and having a rotatingItem property.